src/pages/talks/react18-concurrency/index.tsx

```diff
@@ -0,0 +1,182 @@
+// TODO: move title, description and article link to a single constant
+
+const React18ConcurrencyPage = ({ data }: React18ConcurrencyPageProps) => {
+  return (
+    <Layout>
+      <WidthWrapper>
+        <Helmet>
+          <title>React 18 Concurrency, Explained | PerfPerfPerf</title>
+          {/* Hooray! Meta tags! */}
+          <meta name="description" content="" />
+          <meta name="image" content={fullSocialCoverUrl} />
+          <meta itemProp="name" content="React 18 Concurrency, Explained" />
+          <meta itemProp="description" content="" />
+          <meta itemProp="image" content={fullSocialCoverUrl} />
+          <meta name="twitter:card" content="summary_large_image" />
+          <meta
+            name="twitter:title"
+            content="React 18 Concurrency, Explained"
+          />
+          <meta name="twitter:description" content="" />
+          <meta name="twitter:site" content="@3perfcom" />
+          <meta name="twitter:creator" content="@iamakulov" />
+          <meta name="twitter:image:src" content={fullSocialCoverUrl} />
+          <meta name="og:title" content="React 18 Concurrency, Explained" />
+          <meta name="og:description" content="" />
+          <meta
+            property="og:url"
+            content="https://3perf.com/talks/react18-concurrency/"
+          />
+          <meta property="og:image" content={fullSocialCoverUrl} />
+          <meta property="og:site_name" content="PerfPerfPerf" />
+          <meta property="fb:admins" content="100002052594007" />
+          <meta property="og:type" content="article" />
+          <meta property="article:author" content="Ivan Akulov" />
+          <script type="application/ld+json">
+            {JSON.stringify({
+              '@context': 'https://schema.org',
+              '@type': 'TechArticle',
+              headline: 'React 18 Concurrency, Explained',
+              description: '',
+              image: fullSocialCoverUrl,
+              author: {
+                '@type': 'Person',
+                name: 'Ivan Akulov',
+                url: 'https://twitter.com/iamakulov',
+              },
+              publisher: {
+                '@type': 'Organization',
+                name: 'PerfPerfPerf',
+                url: 'https://3perf.com',
+                logo: {
+                  '@type': 'ImageObject',
+                  url: 'https://3perf.com/3perf-logo-black-raster.png',
+                  width: 1500,
+                  height: 1500,
+                },
+              },
+              datePublished: publishedDate.toISOString(),
+              dateModified: lastUpdatedDate.toISOString(),
+              mainEntityOfPage: {
+                '@type': 'WebPage',
+                url: 'https://3perf.com/talks/react18-concurrency/',
+              },
+            })}
+          </script>
+          {/* // ? Do we even need this? (copy-pasted from web-perf-101) */}
+          <meta property="foo" content="test" />
```

Comment:
Do we even need this?

Reply:
Darn, I’m not even sure why it’s there. Safe to remove, thank you 🙌